uuid/mod.ts

```diff
@@ -44,7 +44,7 @@ import { generate as generateV5, validate as validateV5 } from "./v5.ts";
  * import { assert } from "@std/assert/assert";
  *
  * const uuid = v1.generate();
- * assert(v1.validate(uuid));
+ * assert(v1.validate(uuid as string));
```

Comment:
It seems that when changed, the runtime type-checks this code with a version of `@std/uuid` prior to the version defined in `uuid/deno.json`. This appears to be a bug.